src/game/characters/PlayerCharacter.ts

```diff
@@ -0,0 +1,40 @@
+import { BaseCharacter, BaseStats } from "./BaseCharacter";
+
+export class PlayerCharacter extends BaseCharacter {
+    private name: string;
+    private level: number;
+    sprite: Phaser.Types.Physics.Arcade.SpriteWithDynamicBody;
+
+    constructor(name: string, level: number, stats: BaseStats, sprite: Phaser.Types.Physics.Arcade.SpriteWithDynamicBody) {
+        super(stats);
+        this.name = name;
+        this.level = level;
+        this.sprite = sprite;
+    }
+
+    createAnimations(scene: Phaser.Scene, asepriteKey: string): void {
+        scene.anims.createFromAseprite(asepriteKey);
+    }
```

Comment:
We need to do this on the scene's preload. We're already doing it for the current sprite, and we'll need to do it before we create the player character, so no need for this here.